storage/gcloud/aio/storage/storage.py

```diff
@@ -17,6 +17,11 @@
 from typing import Union
 from urllib.parse import quote
 
+try:
+    from cString import StringIO as BytesIO
```

Comment:
I'm not familiar with this library. By any chance, did you mean to refer to [`cStringIO`](https://docs.python.org/2/library/stringio.html#module-cStringIO) ?

Reply:
As I understand, `str` in Python 2 is equivalent to `bytes` in Python 3, and `unicode` in Python 2 is equivalent to `str` in Python 3. Therefore, `StringIO` in Python 2 is equivalent to `BytesIO` in Python 3. `cStringIO` is only available in Python 2, and import will fail in Python 3. 

But rethinking about this, I think it would be better to only wrap data with when using aiohttp, and skip this with rest api. I will update the PR accordingly.

Reply:
Right, sorry, what I'd meant to call out was the literal name of the import.
```
~ » python2
>>> import cString
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named cString
>>> import cStringIO
>>> 
```

That said, your comment on the other approach makes sense to me, so I'll take a look at reviewing that instead :)